services/generic/fan_control.md

```diff
@@ -1,33 +1,40 @@
-### Fan control service
+# Fan Control Service
 
-The service has to be used to control a fan operational modes, speed and receive state updates.
+The service is used to control a fan operational modes, speed and receive state updates.
```

Comment:
Add note that fan can also be controlled over the Thermostat service, but then it's impossible to set the fan power.